operator/src/main/java/org/keycloak/operator/Constants.java

```diff
@@ -42,7 +42,8 @@ public final class Constants {
     public static final Map<String, String> DEFAULT_DIST_CONFIG = Map.of(
         "health-enabled","true",
         "cache", "ispn",
-        "cache-stack", "kubernetes"
+        "cache-stack", "kubernetes",
+        "proxy", "passthrough"
```

Comment:
@vmuzikar One thing. What's the intention behind this? Is there any documentation for it?

Reply:
@vmuzikar Thanks!